### ISSUE TITLE
MathJax.js not found

### ISSUE THREAD

**Issue by jqdg** [gollum/gollum#1704]
I am running gollum with the `--mathjax` flag, but I can't get any math to render. In the web inspector in the browser, I see that it can't find the `MathJax.js` file. In the page source I see
```html
<script defer src="/gollum/assets/mathjax/MathJax.js%3Fconfig=TeX-AMS-MML_HTMLorMML"></script>
```
but `MathJax.js` is not in the `assets` directory (other JavaScript files are). 

Is there some other setup step that I am not performing in order to get MathJax to work? I tried searching the docs but couldn't find anything relevant.

Output of `gollum --versions` (I tried with both Ruby 3.0 and 2.7, same behavior):
```
Gollum 5.2.1
Running on: x86_64-darwin18 with Ruby version 3.0.0
Using:
rugged 0.99.0
gollum-rugged_adapter 1.0
gollum-lib 5.1
Markdown rendering gem: kramdown
Other renderering gems:
RedCloth 4.3.2
org-ruby 0.9.12
asciidoctor 2.0.12
wikicloth 0.8.3
```

**Comment by dometto** [comment]
Confirmed bug, fix in #1706. Will merge and do a bugfix release ASAP, after which `gem update gollum` should fix the problem.

**Comment by dometto** [comment]
Resolved in `v5.2.2`
